helper/dbtxn/dbtxn.go

```diff
@@ -0,0 +1,109 @@
+package dbtxn
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"strings"
+)
+
+type Config struct {
```

Comment:
We could get rid of this object and pass a map in instead. That way we don't have to map the variables to strings below.

Reply:
It started as a map in the old code... The nice thing about it this way is we can check values directly without also checking whether they exist in the map. Either way works though.

Reply:
I don't think we actually check any of the values though, and if we use a map most of the `parseQuery` function could be deleted.

Reply:
They're used in `parseQuery` :-)

But, sure, I'm fine either way. I don't mind this change but I don't see it as being meaningfully better than the previous implementation.

Reply:
Since BK does seem to have stronger feelings than me about this, should probably just revert to using a map.